index.js

```diff
@@ -92,11 +97,15 @@ module.exports = function (sails) {
             var modelDef, modelName, modelClass, cm, im, connectionName,
                 sequelizeMajVersion = parseInt(Sequelize.version.split('.')[0], 10);
 
+            // Try to read settings from old Sails then from the new.
+            // 0.12: sails.config.models.connection
+            // 1.00: sails.config.models.datastore
+            var defaultConnection = sails.config.models.connection || sails.config.models.datastore || 'default';
             for (modelName in models) {
                 modelDef = models[modelName];
                 sails.log.verbose('Loading model \'' + modelDef.globalId + '\'');
 
-                connectionName = modelDef.options.connection || sails.config.models.connection;
+                connectionName = modelDef.options.connection || defaultConnection;
```

Comment:
Don't you forget about model definition datastore variant for v1.0? :)

Reply:
No, it in :103 line.

Reply:
I mean is `modelDef.options.connection` correct? Or it should be `modelDef.options.datastore` ?

Reply:
Oh, I see. You're right. I forgot `modelDef.datastore`